tests/unit/providers/test_anthropic_messages.py

```
@@ -133,6 +181,490 @@ async def test_amessages_non_streaming() -> None:
+    finally:
+        await http_client.aclose()
+
+    assert requests[0].url.query == b"beta=true"
+    assert requests[0].headers["anthropic-beta"] == "fast-mode-2026-02-01"
+
+
+@pytest.mark.asyncio
+async def test_amessages_beta_extra_header_suppresses_unknown_edit_warning(
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    async def handler(request: httpx.Request) -> httpx.Response:
```

Ruff (0.16.0)

[warning] Unused function argument: `request`

(ARG001)